README.md

```diff
@@ -16,17 +16,41 @@ Please [view our contribution guidelines](https://github.com/kabanero-io/kabaner
 Please see [Contributing to the blog](https://github.com/kabanero-io/blogs) in our Blogs repository.
 
 ## Contributing to the guides
-Each guide resides in its own repository and is dynamically pulled into the kabanero.io build process through the `scripts/build.sh` shell script. The content of the guide can be written in HTML, markdown, or AsciiDoc formats. The following front matter variables must be set:
+Each guide resides in its own repository and is dynamically pulled into the kabanero.io build process through the `scripts/build.sh` shell script. The content of the guide can be written in HTML, markdown, or AsciiDoc formats- AsciiDoc is preferred. The following front matter variables must be set:
+```
```

Comment:
thought the build_clone_guides.rb script did this

Reply:
actually the build_jekyll_maven script does this, but the entry point is build.sh. I guess its irrelevant anyways. I will just remove the reference